secret.go

```diff
@@ -57,8 +57,26 @@ const (
 //   ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*()-
 //
 // RandomPassword does not support Handler (user-invoked password rotation),
-// it only supports rotation by Secrets Manager.
-type RandomPassword struct{}
+// it only supports rotation by Secrets Manager. The password generated by
+// RandomPassword may be configured by providing RandomPasswordOptions on
+// initialization.
+type RandomPassword struct {
+	Opts *RandomPasswordOptions
+}
+
+// RandomPasswordOptions is used to configure a RandomPassword
+type RandomPasswordOptions struct {
+	// PasswordLength defines the length of the password generated by RandomPassword.
+	// If not provided, DEFAULT_PASSWORD_LENGTH will be used
```

Comment:
I don't see `DEFAULT_PASSWORD_LENGTH` defined/declared in this PR? Missing a file?

Reply:
already existed but unused heh https://github.com/square/password-rotation-lambda/blob/master/secret.go#L49